vulnerable_compilable.cpp

```
@@ -0,0 +1,108 @@
+public:
+    VulnerableServer() {
+        adminUser = "admin";
+        adminPass = "password123";
+    }
+
+    // 模擬一個綜合性服務流程
+    void run() {
+        // 從檔案讀取指令，但未檢查檔案是否存在、也沒有驗證輸入長度
+        char inputCommand[10] = {0};
+        ifstream infile("/nonexistent_file.txt");
+        infile.getline(inputCommand, 50);  // 讀取超過 inputCommand 陣列大小
```

Cppcheck (2.10-2)

[error] Buffer is accessed out of bounds

(bufferAccessOutOfBounds)